.travis.yml

```diff
@@ -286,6 +286,7 @@ before_deploy:
           rm -rf obj/build/dist/doc &&
           cp -r obj/build/dist/* deploy/$TRAVIS_COMMIT;
       fi
+  - gem update --system
```

Comment:
This presumably accesses network, so needs to be put in a retry loop of some sort.

Reply:
Okay, will add a `travis_retry` after CI stops being yellow.